packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -184,6 +201,8 @@ class IconButton extends StatelessWidget {
 
   /// The splash radius.
   ///
+  /// If [ThemeData.useMaterial3] is set to true, this will not be used.
```

Comment:
My natural question as a developer is, what do I use to specify splash radius? Please add a little more information to these comments.